activity_calendar/models.py

```diff
@@ -79,10 +80,12 @@ class Meta:
 
     # The date at which the activity will become visible for all users
     published_date = models.DateTimeField(default=now_rounded)
+    is_public = models.BooleanField(default=True, help_text="If activity should be on public calendar")
```

Comment:
```suggestion
    is_public = models.BooleanField(default=True, help_text="If activity should be on the public calendar (outside of Squire)")
```

Reply:
Should also apply inside of Squire. Updated that now.